### PR TITLE
New copy for members search results

### PR DESCRIPTION
**Pull request by julioromano**
Context: https://matrix.to/#/!dypRwVXIkJaTAACHPd:element.io/$Obt7MbgGtHBqJHpZqcdd6OQ5XgxxNXSJofnFG1NyWWM?via=matrix.org&via=element.io&via=t2l.io

Related to: https://github.com/vector-im/element-x-android/issues/246

### CODE REVIEW

File: features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt [hunk 1]
```diff
@@ -71,6 +71,7 @@ import io.element.android.libraries.matrix.ui.components.MatrixUserRow
 import kotlinx.collections.immutable.ImmutableList
 import io.element.android.libraries.ui.strings.R as StringR
 
+@OptIn(ExperimentalMaterial3Api::class)
```
Comment (Florian14) [inline features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt:74]: According to the linter, there is no experimental API

File: features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt [hunk 2]
```diff
@@ -149,6 +150,29 @@ private fun RoomMemberList(
     }
 }
 
+@Composable
+private fun RoomMemberSearchResult(
```
Comment (Florian14) [inline features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt:154]: I think you wanted to rename the method but the previous one is still present

Comment (Florian14) [reply features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt:154]: Would we have a single method with a param for the header title?

File: features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt (hunk 1 above)
Comment (julioromano) [reply features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt]: Indeed, my AS was out of sync.

File: features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt (hunk 2 above)
Comment (julioromano) [reply features/roomdetails/impl/src/main/kotlin/io/element/android/features/roomdetails/impl/members/RoomMemberListView.kt]: Refactored

**Comment by julioromano** [comment]
All suggestion addressed!

**Review by Florian14 — APPROVED** [review]
SGTM

**Pull request merged**